posthog/clickhouse/migrations/0069_add_channel_definitions.py

```diff
@@ -0,0 +1,8 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.channel_type.sql import (
+    CHANNEL_DEFINITION_DATA_SQL,
+)
+
+operations = [
+    run_sql_with_exceptions(CHANNEL_DEFINITION_DATA_SQL),
```

Comment:
It looks like this will create duplicates in this table - not sure if that's desired, but nothing else looks off so :ship: it!